### ISSUE TITLE
When new docx document is created, it can´t be edited.

### ISSUE THREAD

**Issue by taavit21** [ONLYOFFICE/onlyoffice-nextcloud#553]
Hello,

So i noticed this issue.
I don´t know whetever it´s releated to the recent nextcloud upgrade to 22.2.3 or ONLYOFFICE 7.2.0 but i am unable to write anything at all when:
![1](https://user-images.githubusercontent.com/47745930/142566598-8e24b8c4-db1b-4c2e-967e-b4caabf9823c.PNG)

![2](https://user-images.githubusercontent.com/47745930/142566605-c20181a2-dc61-4daf-a36c-3c7a0a731169.PNG)

**Comment by SergeyKorneyev** [comment]
Hello,
Could you specify your Document Server version?

**Comment by taavit21** [comment]
Sure.

```
supervisor> version
4.1.0
supervisor>
```

**Comment by ShockwaveNN** [comment]
@taavit21 This is version of `supervisor` lib we use to control our services, not of DocumentServer itself

If you use docker or deb installation - use `dpkg -l | grep onlyoffice-documentserver`

**Comment by taavit21** [comment]
Ah, sure.

```
root@onlyoffice:~# dpkg -l | grep onlyoffice-documentserver
ii  onlyoffice-documentserver             6.4.2-6                               amd64        Online editors for text documents, spreadsheets, and presentations
```

**Comment by ShockwaveNN** [comment]
Good, this is latest version indeed

In that case my colleague continue to investigate your issue

**Comment by SergeyKorneyev** [comment]
@taavit21 
Could you check which display mode for collaborative changes you have enabled? It's possible that you have the Final mode enabled either in the editor interface or in Nextcloud options:

![photo_2021-12-09_17-53-36](https://user-images.githubusercontent.com/77327210/145548630-ba3b8c67-26f9-4da4-b404-81cd5e129ae3.jpg)


![chrome_2021-12-10_12-15-08](https://user-images.githubusercontent.com/77327210/145548877-4666991c-0877-40fb-9ad0-54c81c6b521a.png)

**Comment by taavit21** [comment]
Hello,

This issue was resolved with onlyoffice 7.2.1
